src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_MAX_SIZE = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        this.keys = (K[]) new Object[ARRAY_MAX_SIZE];
+        this.values = (V[]) new Object[ARRAY_MAX_SIZE];
```

Comment:
no need in `this` keyword since there are no other variables with similar names in scope of this method
```suggestion
        keys = (K[]) new Object[ARRAY_MAX_SIZE];
        values = (V[]) new Object[ARRAY_MAX_SIZE];
```